Library/Homebrew/dev-cmd/tap-new.rb

```diff
@@ -85,10 +85,6 @@ def self.tap_new
                 key: ${{ runner.os }}-rubygems-${{ steps.set-up-homebrew.outputs.gems-hash }}
                 restore-keys: ${{ runner.os }}-rubygems-
 
-            - name: Install Homebrew Bundler RubyGems
```

Comment:
Isn't this needed any more? Why have the cache but not the install?

Reply:
`brew test-bot` should handle this, as it's just a series of `brew test-bot` commands.

Reply:
> The `cache-hit` condition is also incorrect as that won't detect when gems need updating due to the relaxed restore key, so in reality it never was running after the initial caching run until `brew test-bot` does it itself.

Gotcha, this is the bit I was worried about.